examples/docker/mssql/entrypoint.sh

```diff
@@ -2,26 +2,36 @@
 
 set -e
 
+INITDB_FOLDER="/opt/mssql-scripts/"
+TMP_DIR="/tmp/"
+
+function sqlcmd() {
+  file=${1?"Sql file must be provided as a parameter"}
+  /opt/mssql-tools/bin/sqlcmd -S localhost -U SA -P "${SA_PASSWORD}" -i "${file}" && \
+    echo "$file imported successfully"
+}
+
 function initialize_app_database() {
     # Restore the application database using .sql files
     shopt -s globstar nullglob
-    for file in /opt/mssql-scripts/*.sql; do
-        echo "Importing $file"
-        /opt/mssql-tools/bin/sqlcmd -S localhost -U SA -P "${SA_PASSWORD}" -i "${file}"
+    for file in "$INITDB_FOLDER"/*.sql; do
+        sqlcmd "$file"
     done
 
-    # TODO - Add support for `.sql.gz` and document it on the README.md of the example
     # Restore the application database using .sql.gz files
     shopt -s globstar nullglob
-    for file in /opt/mssql-scripts/*.sql.gz; do
-        echo "Importing $file"
-        echo "Warning: .sql.gz files are not supported yet. Skipping"
-        break
+
+    for file in "$INITDB_FOLDER"/*.sql.gz; do
+        echo "Uncompressing $file to $TMP_DIR"
+        filename=$(basename "$file")
+        uncompressed_file="$TMP_DIR/${filename//.gz/}"
+        #TODO mkfifo support?
```

Comment:
```suggestion
        # TODO Add support for mkfifo on mssql entrypoint
```
this will automatically create a Github issue once it is merged